spec/controllers/api/v1/skills_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe Api::V1::SkillsController, type: :controller do
+
+  let(:skill) { create(:skill) }
+
+  describe "index" do
+
+    before do
+      FactoryGirl.reload
+      2.times { create(:skill, featured: true) }
+      2.times { create(:skill) }
+    end
+
+    it 'returns a json array of skills' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.